src/ui/server/helpers/strip-empty-form-fields/index.ts

```diff
@@ -16,7 +17,13 @@ const stripEmptyFormFields = (formBody: RequestBody) => {
 
     const hasValue = Boolean(value) || value === false;
 
-    if (hasValue) {
+    /**
+     * if array exists and if key is in array
+     * then keeps the key and value pair in form data - even if empty
+     */
+    const shouldBeNullOrEmptyString = nullOrEmptyStringFields && nullOrEmptyStringFields.includes(key);
```

Comment:
Should we use `isPopulatedArray` here? 

```js
isPopulatedArray(nullOrEmptyStringFields) && nullOrEmptyStringFields.includes
```

Reply:
will not work because eslint will say that nullOrEmptyStringFields could be undefined with a linting error